storage-bigtable/src/bigtable.rs

```diff
@@ -215,6 +206,23 @@ impl BigTableConnection {
         }
     }
 
+    pub fn new_for_emulator(
+        instance_name: &str,
+        app_profile_id: &str,
+        endpoint: &str,
+        timeout: Option<Duration>,
+    ) -> Result<Self> {
+        Ok(Self {
+            access_token: None,
+            channel: tonic::transport::Channel::from_shared(format!("http://{}", endpoint))
```

Comment:
is tls available?

Reply:
I think it could be but I would like only support http atm.